src/app/pages/directory-service/directory-services.elements.ts

```diff
@@ -13,9 +13,5 @@ export const directoryServicesElements = {
       synonyms: [T('Active Directory')],
       anchor: 'configure-active-directory',
     },
-    configureLdap: {
-      hierarchy: [T('Configure LDAP')],
-      anchor: 'configure-ldap',
-    },
```

Comment:
This looks strange. Why is this file used for? Only for the search? If so why LDAP was removed and why AD is still here?

Reply:
@william-gr   This is used for search autocompletion. I removed it since it was not used. I'll check if others are used as well 👍